src/types/index.ts

```diff
@@ -5,5 +5,8 @@ export * from './button'
 export * from './card'
 export * from './catalog'
 export * from './code-block'
+export * from './collapse'
+export * from './data-time-picker'
```

Comment:
This file is named incorrectly; it should be this
```suggestion
export * from './date-time-picker'
```